pyproject.toml

```diff
@@ -81,7 +81,6 @@ test = [
     "black==22.6.0",
     "docutils",
     "ipykernel",
-    "jupyter-contrib-nbextensions",
```

Comment:
@opotowsky I thought this was needed to build the docs.

IF so, then we should move it down to the "docs" dependencies list, not delete it.

Can you check on that?

Reply:
I just want to check that our jupyter notebooks in the docs still build and run, after this change you made.

Reply:
Done! https://github.com/terrapower/armi/pull/1567/commits/eaf635557f9ab3dc5a50dacb3f5e8c4c6440cfb3